routers/commentsRouter_controllers.go

```diff
@@ -0,0 +1,28 @@
+package routers
+
+import (
+	beego "github.com/beego/beego/v2/server/web"
+	"github.com/beego/beego/v2/server/web/context/param"
+)
+
+func init() {
+
+    beego.GlobalControllerRouter["go-google-scraper-challenge/controllers:UserController"] = append(beego.GlobalControllerRouter["go-google-scraper-challenge/controllers:UserController"],
+        beego.ControllerComments{
+            Method: "New",
+            Router: "/",
+            AllowHTTPMethods: []string{"get"},
+            MethodParams: param.Make(),
+            Filters: nil,
+            Params: nil})
+
+    beego.GlobalControllerRouter["go-google-scraper-challenge/controllers:UserController"] = append(beego.GlobalControllerRouter["go-google-scraper-challenge/controllers:UserController"],
+        beego.ControllerComments{
+            Method: "Post",
+            Router: "/",
+            AllowHTTPMethods: []string{"post"},
+            MethodParams: param.Make(),
+            Filters: nil,
+            Params: nil})
```

Comment:
Not sure why this format differs so much from what Hoang did: https://github.com/hoangmirs/go-scraper/pull/23/files#diff-4bb1f3889791d48e3a1866a36aca485a8bf94a73d02e0950f32b69c8004edd2a

Reply:
@olivierobert This file is auto-generated by `Beego` after running `bee run` not sure that we should ignore it or not 🤔